.github/scripts/lint-pr-title.cjs

```diff
@@ -0,0 +1,125 @@
+/**
+ * This Script can be tested locally with
+ *
+ * export GH_TOKEN=**** GITHUB_REPOSITORY=quisquous/cactbot PR_NUMBER=$NUM
+ * node ./.github/scripts/lint-pr-title.cjs
+ */
+'use strict';
+
+const github = require('@actions/github');
+
+const botName = 'cactbotbot';
+
+const validScope = [
+  'resources',
+  'ui',
+  'pullcounter',
+  'raidboss',
+  'raidemulater',
+  'oopsy',
+  'radar',
+  'eureka',
+  'jobs',
+  'test',
+  'util',
+  'config',
+  'lint',
+  'build',
+  'ci',
+  'i18n',
+];
+
+const validNonScope = [
+  '[WIP]',
+  '[wip]',
+  '!', // means `BREAKING CHANGE!`
+];
+
+const thanksComment = (userName) => {
+  let userStr = '';
+  if (userName)
+    userStr = `@${userName} `;
+  return `${userStr}Thanks for your contribution!🚀`;
+};
+
+const getComment = (title, userName) => `${thanksComment(userName)}
+
+Currently your title is: ${title},
+but it should be in the format of \`(nonscope) scope: description\`.
+
+\`scope\` can be any of the following:
+${validScope.map((s) => `  - ${s}`).join('\n')}
+
+\`nonscope\` can be any of the following (although this is not always required):
+${validNonScope.map((s) => `  - ${s}`).join('\n')}
```

Comment:
I think the non-scope prefix is kind of more of a power user feature, and it'd be better if the bot just gave simpler instructions that are easier to follow for folks who might be new to the project.  What do you think?

```suggestion
Currently your title is: ${title},
but it should be in the format of \`scope: description\`.

\`scope\` can be any of the following:
${validScope.map((s) => `  - ${s}`).join('\n')}
```

Reply:
Okay!